docs/source/getting-started/setting-up-virtualization-environment.adoc

```diff
@@ -0,0 +1,301 @@
+include::variables.adoc[]
+
+[[setup-virtualization-environment]]
+= Setting Up the Virtualization Environment
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+:next-steps: Proceed to xref:../getting-started/installing.adoc#[Installing {project}] once your hypervisor has been installed and configured.
+
+toc::[]
+
+[[setup-virtualization-environment-overview]]
+== Overview
+
+Follow the appropriate procedure to set up virtualization for your particular operating system and hypervisor.
```

Comment:
Mentioning both virtualization and hypervisor in the same sentence confuses the meaning. The sentence should be `Follow the appropriate procedure to set up hypervisor for your particular operating system.`